### ISSUE TITLE
Can't link TFLite Ops in xcode anymore

### ISSUE THREAD

**Issue by andrew-lyons** [tensorflow/tensorflow#62972]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): 
    - Macbook M1 Pro
    - XCode version 15.0.1
- TensorFlow installed from (source or binary): 
```
  pod 'TensorFlowLiteSwift', '~> 2.14.0', :subspecs => ['Metal']
  pod 'TensorFlowLiteSelectTfOps', '~> 0.0.1-nightly'
```

**Standalone code to reproduce the issue** 
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.

Also, please include a link to a GraphDef or the model if possible.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem.
If including tracebacks, please include the full traceback. Large logs and files
should be attached.

---

I'm not sure if a reproducible example is necessary, perhaps I'm wrong. The documentation says to include the TFLite Select Ops via adding a linker flag to 'Other Linker Flags' in the app's Build Settings. This results in the following:

```
Unknown argument: '-force_load /(my local path)/ios/App/Pods/TensorFlowLiteSelectTfOps/Frameworks/TensorFlowLiteSelectTfOps.xcframework/ios-arm64/TensorFlowLiteSelectTfOps.framework/TensorFlowLiteSelectTfOps'
```

Is there a different method for linking the select ops? As listed above this is on XCode version 15.0.1.

**Comment by LakshmiKalaKadali** [comment]
Hi @andrew-lyons,

Try to specify both `TensorFlowLiteSwift` and  `TensorFlowLiteSelectTfOps` are the same nightly versions in `pod` file target.
```
  # Specify in your pod file target
  pod 'TensorFlowLiteSwift', '~> 0.0.1-nightly'
  pod 'TensorFlowLiteSelectTfOps', '~> 0.0.1-nightly'

```
Thank You

**Comment by andrew-lyons** [comment]
@LakshmiKalaKadali I changed them both to nightly builds, and still receive the same issue:

```
INFO: Initialized TensorFlow Lite runtime.
TensorFlow Lite Error: Select TensorFlow op(s), included in the given model, is(are) not supported by this interpreter. Make sure you apply/link the Flex delegate before inference. For the Android, it can be resolved by adding "org.tensorflow:tensorflow-lite-select-tf-ops" dependency. See instructions: https://www.tensorflow.org/lite/guide/ops_select
TensorFlow Lite Error: Node number 1 (FlexMutableHashTableV2) failed to prepare.
```

I am unable to apply the linker flag in the tflite documentation [here](https://www.tensorflow.org/lite/guide/ops_select) because of the linker error I pasted in my original post body. Perhaps xcode 15 does not allow the usage of `-force-load`?

**Comment by pkgoogle** [comment]
Hi @andrew-lyons, I believe you are correct that 15.0.1 does not allow 'force_load', though we are not xcode experts so you might want to check with them just in case. Another alternative is to see if you can reduce your version of xcode so that you may continue your work.

@yishuangP can you please take a look? Thanks.

**Comment by andrew-lyons** [comment]
@pkgoogle @yishuangP I'm attempting to downgrade xcode, however being on macOS Sonoma means that xcode 14 is not officially supported for use. Is there any other way to link the select ops other than using the `Other Flags`?

**Comment by andrew-lyons** [comment]
@yishuangP @pkgoogle is there anything on this end that can be done for this? It seems tflite is unusable currently with these newest versions.

**Comment by robin-torwell** [comment]
Hey @andrew-lyons,

I just ran into this exact issue myself. Turns out you need to add

`-force-load`

and 

`/(your local path)/ios/App/Pods/TensorFlowLiteSelectTfOps/Frameworks/TensorFlowLiteSelectTfOps.xcframework/ios-arm64/TensorFlowLiteSelectTfOps.framework/TensorFlowLiteSelectTfOps`

each in a separate line. Otherwise it get's double quoted an this messes things up.

**Comment by BasilZinsli** [comment]
Hi Guys

it has to be 

-force_load 

instead of 

-force-load

So yes this works in terms of that the linker is found but i still get the error:

TensorFlow Lite Error: Select TensorFlow op(s), included in the given model, is(are) not supported by this interpreter. Make sure you apply/link the Flex delegate before inference. For the Android, it can be resolved by adding "org.tensorflow:tensorflow-lite-select-tf-ops" dependency. See instructions: https://www.tensorflow.org/lite/guide/ops_select
TensorFlow Lite Error: Node number 41 (FlexTranspose) failed to prepare.

**Comment by andrew-lyons** [comment]
@robin-torwell Thanks for that, just getting back around to this. I was able to move forward using your comment 😄 

Unfortunately, getting another issue:
```
TensorFlow Lite Error: Op type not registered 'RegexSplitWithOffsets' in binary running on localhost. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib (e.g. `tf.contrib.resampler`), accessing should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.
```

So looking into this now, but I suppose for this issue we could call this taken care of.
